src/legacy/core_plugins/kibana/public/visualize/index.js

```diff
@@ -29,8 +29,8 @@ import { VisualizeConstants } from './visualize_constants';
 import { FeatureCatalogueRegistryProvider, FeatureCatalogueCategory } from 'ui/registry/feature_catalogue';
 import { getLandingBreadcrumbs, getWizardStep1Breadcrumbs } from './breadcrumbs';
 
-import { data } from 'plugins/data/setup';
-data.filter.loadLegacyDirectives();
+// load directives
+import '../../../data/public';
```

Comment:
nit: these might be more readable as `../../../../core_plugins/data/public` since `core_plugins` is a more recognizable directory for devs familiar with our repo.

Reply:
Or perhaps even better, the approach outlined here: https://github.com/elastic/kibana/issues/40446

Reply:
It's great, but as far as I can tell, it's not ready yet. Right?
What do you say we make a separate PR for this?

Reply:
i _think_ the absolute imports will work client side today, at least in OSS code

Reply:
a separate PR works for me too; i only mentioned this here since you're already going to the trouble of updating all of the imports and this could help future proof them.

but it isn't critical to address this in the scope of this PR; we could always do one big update later once #40446 has an implementation merged

Reply:
Done